camelot/model/party.py

```diff
@@ -316,6 +316,14 @@ class City(GeographicBoundary, WithCountry):
 
     __mapper_args__ = {'polymorphic_identity': 'city'}
 
+    def __init__(self, *args, **kwargs):
+        self._reconstruct()
+        super().__init__(*args, **kwargs)
+
+    @orm.reconstructor
+    def _reconstruct(self):
+        self._zipcode_validator = ZipcodeValidator()
```

Comment:
zouden we voor deze purpose de ZipCodeValidator nt opsplitsen in een pure python object, en een QValidator, want nu heb je de constructie van een QValidator per city object in de session

Reply:
Verder gaande op daarstraks, eigenlijk hebben we dat python object al in de vorm van de state.
Dus ik kan de inner State klasse toplevel maken en daar de python functionaliteit naar verhuizen.
En dan kan de state op de city en address instances worden bijgehouden ipv de validator.

Reply:
lijkt me idd prima ... en dan is die state class gewoon serialiseerbaar ...

Reply:
Allright, toegepast in https://github.com/v-finance/vFinance/pull/2881 en https://github.com/v-finance/camelot/pull/613.